client/state/selectors/get-reader-follows.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { filter } from 'lodash';
+
+/*
+ * Get all sites/feeds the user follows.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Integer} Follow count
+ */
+export default function getReaderFollows( state ) {
+	return filter( state.reader.follows.items, [ 'is_following', true ] );
```

Comment:
this _really_ needs to memoized. might need to return all of the items too, depending on where we want to stash the ui state we talked about last week.